docs/UI/OnLanguageChange/index.md

```diff
@@ -24,6 +24,10 @@ OnLanguageChange
 ![OnLanguageChange](./assets/OnLanguageChange.png)
```

Comment:
this diagram says after the user switches the language, HMI sends some `UI.LanguageChange` 
currently, I don't see this function in HMI_API.
It might be worth to merge these two diagrams into 1, since overwriting and caching doesn't happen without language change, does it?

Reply:
Well noted, thank you.
Fixed in 747f717cb19236077dd65e3871b93ca960402f5c